java/connector-node/risingwave-sink-jdbc/pom.xml

```diff
@@ -52,8 +56,21 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.apache.derby</groupId>
-            <artifactId>derby</artifactId>
+            <groupId>org.testcontainers</groupId>
```

Comment:
May declare these dependencies in root pom and reference here?

Reply:
For the test dependency I think we can declare them here directly.